.gitignore

```diff
@@ -21,3 +21,6 @@ package-lock.json
 !.yarn/sdks
 !.yarn/versions
 .pnp.*
+/scripts/tools/eslint-plugin-prettier-internal-rules/node_modules
+/scripts/tools/eslint-plugin-prettier-internal-rules/.nyc_output
+/scripts/tools/eslint-plugin-prettier-internal-rules/yarn.lock
```

Comment:
Why?

Reply:
It's only needed when testing, and the CI only run when `/scripts/tools/eslint-plugin-prettier-internal-rules` changed, no need store it. And it will run install when running test

https://github.com/prettier/prettier/pull/9929/files#diff-1d4b23fb60ed2d3893253d77b42c31970f670eddcf3ea4d4403ee50a145e8de7R10

Reply:
And it even not needed if I don't collect coverage(with `nyc`), I'm adding these devDependencies, so we can skip `yarn install` on project root when tesing on CI.

Reply:
Anyway, if you don't like it, I'll remove devDependencies and use `npx nyc node test.js` to run test and collect coverage.

Reply:
Already did https://github.com/prettier/prettier/pull/9929/commits/a41c8d6cc2cd72ecf26fe3acdb3df9821b8501ea, so we are both happy :smile:

Reply:
Oh sorry, It was just a question. I wondered why `yarn.lock` isn't committed. I think we should have devDependencies for internal ESLint rules because that is an another package.

Reply:
> I think we should have devDependencies

No, there is no extra dependencies needed now.

Reply:
I see